src/PhpSpreadsheet/Calculation/Calculation.php

```diff
@@ -2874,9 +2869,11 @@ public static function getExcelConstants(string $key): bool|null
         ],
     ];
 
-    public function __construct(?Spreadsheet $spreadsheet = null)
-    {
-        $this->spreadsheet = $spreadsheet;
+    public function __construct(/**
+         * Instance of the spreadsheet this Calculation Engine is using.
+         */
+        private ?Spreadsheet $spreadsheet = null
+    ) {
```

Comment:
This isn't great. It looks ugly, even more so with multiple parameters, and it is not supported by by IDE AFAIK. Instead it should be a proper `@param` annotation IMHO.